cli/src/command/account.rs

```diff
@@ -644,8 +664,19 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| {
+                a.metadata
+                    .milestone_timestamp_booked()
+                    .cmp(&b.metadata.milestone_timestamp_booked())
+            });
```

Comment:
Just asking, do we really want the most recent outputs first ?

Reply:
good question ... but I feel like if we answer this question with yes for transactions (and currently we do) I would probably say yes for the outputs as well.